requirements-dev.txt

```diff
@@ -8,5 +8,5 @@ numpydoc>=0.9.0
 scikit-learn>=0.22.1
 xgboost>=1
 nox
-lightgbm>=2.3.0
+lightgbm==3.2.1
```

Comment:
This isn't necessary, we already installed 3.x in testing.

We need to update `setup.py` though to change `lightgbm>=2,<2.4` to `lightgbm>=2,<4`